recipe/meta.yaml

```diff
@@ -16,18 +16,20 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
     - cmake
     - ninja
     - pkg-config
     - libgrpc
+    - zlib
```

Comment:
Can you remove these two libraries here? They should only be necessary in `host:`.

Reply:
Ah true; if any plugins need to be executed, then we need to match the architecture of the build host. Could you add a selector `# [build_platform != target_platform]` please, so that this becomes clearer going forward?